### DISCUSSION TITLE
Community request: PS2 videos, OPL Manager, VMC downloads, OPL themes/art, etc

### DISCUSSION THREAD

**Discussion by danmons** [retronas/discussions#104] [Q&A]
Samba + OpenPS2Loader works well, however by default has no artwork included.

OPL Manager is a piece of software designed to configure and customise how OPL looks, with theming, artwork, screenshots, etc:
https://oplmanager.com/site/

And in theory should work if pointing to a SMB/CIFS UNC or mapped drive.

A video guide on how to do that would be really helpful to new users.  Likewise `ART` (artwork), `VMC` (Virtual Memory Cards), `CHD` (cheats/hacks), etc packs (to go in the matching folders that RetroNAS creates), for example:
https://www.psx-place.com/threads/opl-all-in-one-configuration-thread-art-cfg-cht-thm-and-vmc-packs-for-all-games-hdd-usb-smb.28204/

If anyone from the general public wants to contribute a video guide, that would be very helpful. 

Also, if someone wants to make a RetroNAS themed art pack we can deploy by default, that would be very cool.

**Comment by sairuk** [comment]
I think the PS2 space _in general_ could do with some clean **documentation** and guides.

It took me a lot of forum threads and versions of software to get a working solution when I looked at it a while back. Particularly when the term NBD was never clarified at the get go and hdldump support was deprecated in later openps2loader builds.

Integration of the ps2 tools with the share is certainly in scope for RetroNAS project, but I would just mention to be mindful of scope creep.

some notes 
- https://mameau.com/gaming/ps1onps2/
- https://mameau.com/gaming/ps2hdd/

Something like this would also come into play if we added pfsfuse/nbd storage support (on my list), perhaps not _this_ tool specifically as it is windows only and wont run on the pi very well (if at all) but something not unlike this tool would be great, perhaps a new web based solution. 

I have a pfsfuse + nbd mount setup here I run to manage my ps2 internal hdd, I only target these partitions currently

```
    # partitions
    mount_part __common
    mount_part __.POPS
    mount_part +OPL
    ;;
```

I recall OPLManager under wine was painful and wouldn't work 100% iirc, I need to revisit it one of these days as it is the best tool available at the moment
